Libraries/Utilities/AlertIOS.js

```diff
@@ -27,61 +26,160 @@ export type AlertButtonStyle = $Enum<{
   'destructive': string;
 }>;
 
+type ButtonsArray = Array<{
+  text?: string;
+  onPress?: ?Function;
+  style?: AlertButtonStyle;
+}>;
+
 /**
- * Launches an alert dialog with the specified title and message.
- *
- * Optionally provide a list of buttons. Tapping any button will fire the
- * respective onPress callback and dismiss the alert. By default, the only
- * button will be an 'OK' button.
+ * The AlertsIOS utility provides two functions: `alert` and `prompt`. All
+ * functionality available through `AlertIOS.alert` is also available in the
+ * cross-platform `Alert.alert`, which we recommend you use if you don't need
+ * iOS-specific functionality.
```

Comment:
No, there isn't currently a cross-platform `Alert.prompt`, although there's no reason why this has to be iOS-specific.